src/python_testing/TC_OpstateCommon.py

```diff
@@ -1259,11 +1270,12 @@ async def TEST_TC_OPSTATE_BASE_2_6(self, endpoint=1):
         sub_handler = ClusterAttributeChangeAccumulator(cluster)
         await sub_handler.start(self.default_controller, self.dut_node_id, endpoint)
 
+        self.step(3)
         if self.pics_guard(self.check_pics(f"{self.test_info.pics_code}.S.M.ST_RUNNING")):
-            self.step(3)
             self.send_manual_or_pipe_command(name="OperationalStateChange",
                                              device=self.device,
-                                             operation="Start")
+                                             operation="Start",
+                                             msg="Put the DUT into the running state")
```

Comment:
```suggestion
                                             msg="Put DUT in running state")
```

To be consistent with the similar messages below.